package.json

```diff
@@ -21,28 +22,28 @@
   },
   "license": "MIT",
   "dependencies": {
+    "axios": "^1.7.2",
     "express": "^4.18.1",
-    "morgan": "^1.10.0",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "^9.0.2",
```

Comment:
`jsonwebtoken` version 9.0.0 and above introduces a bug, we need to leave it at 8.5.1

Reply:
Has now been reverted, good spot